### PR TITLE
Remove headers from summary

### PR DESCRIPTION
**Pull request by richarddubay**
### What does this PR do, or why is it needed?
This PR removes any headers from the text that could display in the post summary on a card.

**Before:**
Screenshot 2020-03-03 15 54 18

**After:**
Screenshot 2020-03-03 15 55 58

### What design trade-offs/decisions were made?
N/A

### How do I test this PR?
Look at the "This March at NewSpring" content item and make sure that the "Connect Classes" header does not show in the summary.

### What automated tests did you write?
None.

## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes [tag-issues-here]
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android
- [x] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-data-connector-rock/src/content-items/data-source.js [hunk 1]
```diff
@@ -186,8 +186,9 @@ export default class ContentItem extends RockApolloDataSource {
     const tokenizer = new natural.SentenceTokenizer();
     const tokens = tokenizer.tokenize(
       sanitizeHtmlNode(content, {
-        allowedTags: [],
+        allowedTags: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'],
         allowedAttributes: [],
+        exclusiveFilter: (frame) => frame.tag.match(/^(h1|h2|h3|h4|h5|h6)$/),
```
Comment (vinnyjth) [inline packages/apollos-data-connector-rock/src/content-items/data-source.js]: Nice! I can't see a reason not to do this 👍
